src/sentry/snuba/metrics/query.py

```diff
@@ -166,6 +166,8 @@ class MetricsQuery(MetricsQueryValidationRunner):
     # doesn't take into account time bounds as the alerts service uses subscriptable queries that react in real time
     # to dataset changes.
     is_alerts_query: bool = False
+    # Need to skip the orderby validation for ondemand queries
```

Comment:
Can you add another sentence as to way we have to skip it vs. just using existing validation.